conf/mesos-site.xml

```diff
@@ -53,6 +53,16 @@
   </property>
 
   <property>
+    <name>mesos.hdfs.mesosdns</name>
```

Comment:
Why have the boolean instead of just defaulting 'mesos.hdfs.mesosdns.domain' to EmptyString, and then use if (conf.getMesosDnsDomain() != "") instead of conf.usingMesosDns()?